app/src/main/java/com/example/tasky/authentication/presentation/login/viewmodel/LoginViewModel.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.tasky.authentication.presentation.login.viewmodel
+
+class LoginViewModel {
+}
+
+data class LoginState(
+    val isValidEmail: Boolean = false,
+    val isLoading: Boolean = false,
+    val emailValue: String = "",
+    val passwordValue: String = "",
+    val showPassword: Boolean = false
```

Comment:
This is a personal thing:

The ShowPassword (same with OnShowPasswordValueChanged) will only be used to change the TaskyPasswordTextField thingy

You won´t use it anywhere else in the Screen, + usually, that textfield will always have that state and interaction (When icon click, hide/show password)

So this is one of the only cases, in which I rather have this specific state on the TaskyPasswordTextfield, rather than on the ViewModel